### ISSUE TITLE
Crash when width and height specified, no placeholder and put inside FittedBox 

### ISSUE THREAD

**Issue by radzish** [dnfield/flutter_svg#25]
Following snippet:
```dart
    return Flexible(
      child: FittedBox(
        fit: BoxFit.fitWidth,
        child: SvgPicture.asset(
          "icon.svg",
          width: 20.0,
          height: 14.0,
        ),
      ),
    );
```

produces crash:
```
I/flutter ( 8836): Loading AppLocalizations for: en_US
I/flutter ( 8836): ══╡ EXCEPTION CAUGHT BY RENDERING LIBRARY ╞═════════════════════════════════════════════════════════
I/flutter ( 8836): The following assertion was thrown during performLayout():
I/flutter ( 8836): 'package:flutter/src/rendering/box.dart': Failed assertion: line 303 pos 12: 'width > 0.0': is not
I/flutter ( 8836): true.
I/flutter ( 8836): 
I/flutter ( 8836): Either the assertion indicates an error in the framework itself, or we should provide substantially
I/flutter ( 8836): more information in this error message to help you determine and fix the underlying cause.
I/flutter ( 8836): In either case, please report this assertion by filing a bug on GitHub:
I/flutter ( 8836):   https://github.com/flutter/flutter/issues/new
I/flutter ( 8836): 
I/flutter ( 8836): When the exception was thrown, this was the stack:
I/flutter ( 8836): #2      BoxConstraints.constrainSizeAndAttemptToPreserveAspectRatio (package:flutter/src/rendering/box.dart)
I/flutter ( 8836): #3      RenderFittedBox.performLayout (package:flutter/src/rendering/proxy_box.dart:2194:26)
I/flutter ( 8836): #4      RenderObject.layout (package:flutter/src/rendering/object.dart:1570:7)
```

Wrapping SvgPicture with SizedBox or specifying placeholder is SizedBox resolves the issue

**Comment by dnfield** [comment]
Fix should be altering https://github.com/dnfield/flutter_svg/blob/master/lib/svg.dart#L567 to be aware that if `widget.width` or `widget.height` are not null, we should use a `SizedBox` instead of the default `LimitedBox`.

**Comment by dnfield** [comment]
Could still be a problem though if someone specifies a placeholder that doesn't size itself appropriately, but I guess that's on the user?

**Comment by dnfield** [comment]
Fixed in e2175724a5dead48b5bd17ebb3735b93ff1bba73

Released to pub as v0.4.1
